browser/brave_ads/tabs/ads_tab_helper.cc

```diff
@@ -149,10 +149,6 @@ bool AdsTabHelper::IsNewNavigation(
 bool AdsTabHelper::IsErrorPage(content::NavigationHandle* navigation_handle) {
   CHECK(navigation_handle);
 
-  if (!navigation_handle->IsErrorPage()) {
```

Comment:
This check considered error pages with HTML content as not `error`. Examples of such pages are:
https://httpstat.us/500
https://httpstat.us/404